core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -2057,6 +2056,13 @@ final class Stream[+F[_], +O] private[fs2] (private[fs2] val underlying: Pull[F,
       }
     }
 
+  /** Prints each element of this stream to standard out, converting each element to a `String` via `Show`. */
+  def printlns[F2[x] >: F[x], O2 >: O](implicit
+      F: Console[F2],
+      showO: Show[O2] = Show.fromToString[O2]
+  ): Stream[F2, INothing] =
+    covaryAll[F2, O2].map(_.show).foreach(F.println)
```

Comment:
Is this method worth keeping, or is this something easilly accessible by the existing combinators? 

If we keep it, perhaps we should save the middle chunks: 
```suggestion
    foreach( (o: O2) => F.println(o.show))
```